client/image_remove.go

```diff
@@ -2,14 +2,25 @@ package client
 
 import (
 	"encoding/json"
+	"errors"
 	"net/url"
 
+	"strings"
+
 	"github.com/docker/docker/api/types"
 	"golang.org/x/net/context"
 )
 
 // ImageRemove removes an image from the docker host.
 func (cli *Client) ImageRemove(ctx context.Context, imageID string, options types.ImageRemoveOptions) ([]types.ImageDelete, error) {
+	if strings.TrimSpace(imageID) == "" {
+		return nil, errors.New("image name cannot be blank")
+	}
+
+	if _, err := parseNamed(imageID); err != nil {
+		return nil, err
+	}
```

Comment:
Dear @dnephin @vdemeester @thaJeztah ,

I also think there is no need this empty validation for rm, but there was a integration test in "integration-cli/docker_cli_rmi_test.go", like below:
![image](https://cloud.githubusercontent.com/assets/22292664/21789552/dba26ca4-d710-11e6-8f60-db8f6557d261.png)

This test do a empty test, so this TestRmiBlank function is also not needed because this stage is not existed really?